tests/test_cli.py

```diff
@@ -338,9 +338,50 @@ def test_action_invalid(self, cli_runner, project):
             result = cli_runner.invoke(cli.run, ['-a', 'notanaction'])
             assert 'Available actions:' in result.output
             assert 'lint' in result.output
+            assert result.exception
+            assert result.exit_code == 1
+
+    def test_plugin_not_installed(self, cli_runner, project):
+        project.write('.therapist.yml', 'plugins:\n  notsimple: ~')
+
+        with chdir(project.path):
+            result = cli_runner.invoke(cli.run)
+            assert 'Installed plugins:' in result.output
+            assert 'simple' in result.output
```

Comment:
that the list of installed plugins actually has the plugin that is installed (i.e. `simple`).